### PR TITLE
{lang}[GCC/4.9.2] Julia 0.3.7 (WIP)

### PR DESCRIPTION
**Pull request by tvandera**
Julia language easyconfig

### CODE REVIEW

File: easybuild/easyconfigs/j/Julia/Julia-0.3.7-GCC-4.9.2-bare.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name = 'Julia'
+version = '0.3.7'
+versionsuffix = '-bare'
+
+easyblock = 'ConfigureMake'
```
Comment (boegel) [inline easybuild/easyconfigs/j/Julia/Julia-0.3.7-GCC-4.9.2-bare.eb]: this should go first, separated from `name =` with a blank line

**Comment by boegel** [comment]
@tvandera: there's a WIP easyblock for Julia, see https://github.com/hpcugent/easybuild-easyblocks/pull/499

Does that still make sense, you think? Would your easyconfig benefit from that too?

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by tvandera** [comment]
There are two options for Julia: go with the system's libraries for BLAS, LAPACK, FFWT or let the julia build process build its own version. This PR goes with the former, PR #499 goes with the latter. In the end PR #499 seems like the better solution.

**Comment by boegel** [comment]
@tvandera: #499 is on PyQuante?

Also, why would letting Julia build its own BLAS/LAPACK/FFTW be better? It's not easy to do that right, in particular w.r.t. MPI etc. And EasyBuild already knows about how to link to the BLAS/LAPACK/FFTW included in the compiler toolchain being used.

If Julia is highly dependent on BLAS/LAPACK/FFTW, then you should use a full toolchain (e.g. `foss` or `intel`) here rather than `GCC`, possible with extra configure options to make sure Julia picks up the BLAS/LAPACK/FFTW libraries provided by the toolchain.

**Comment by tvandera** [comment]
I agree with previous poster: I'll make this a config linking with existing BLAS/LAPACK/... instead of building its own.

**Comment by boegel** [comment]
@tvandera ping?

**Comment by tvandera** [comment]
Yes. It's still on my stack. Unfortunately, the current EasyConfig we are using works fine for us, but is not suited for general inclusion since we compile BLAS, etc ourselves instead of using the versions from the toolchain...

**Comment by migueldiascosta** [comment]
@tvandera, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**